sellercenter-setting/src/main/java/com/gfg/sellercenter/setting/reader/HttpReader.java

```diff
@@ -0,0 +1,50 @@
+package com.gfg.sellercenter.setting.reader;
+
+import java.net.URL;
+import org.json.JSONObject;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import lombok.AllArgsConstructor;
+import java.nio.charset.StandardCharsets;
+
+@AllArgsConstructor
+public class HttpReader
```

Comment:
Maybe some javadoc? It'll be hard to understand what does this class do exactly from the business perspective

Reply:
I will add a java doc